otel/otel-configuration-of-instrumentation.adoc

```diff
@@ -1,8 +1,8 @@
 :_mod-docs-content-type: ASSEMBLY
-[id="otel-instrumentation"]
-= Configuring and deploying the OpenTelemetry instrumentation injection
+[id="otel-configuration-of-instrumentation"]
+= Configuration of instrumentation for the {OTELShortName}
 include::_attributes/common-attributes.adoc[]
-:context: otel-instrumentation
+:context: otel-configuration-of-instrumentation
```

Comment:
I think the previous filename / ID was better tbh

Reply:
Thank you, I agree. I have two configuration-related files, so I thought I'd make the file names and IDs parallel.

Reply:
Thank you, I made a note for myself to look into this after the upcoming release next week.